docker-compose.yml

```diff
@@ -11,6 +11,11 @@ volumes:
     # need to be accessible in test containers.
 
 x-common-variables:
+  read-only-pulumi-mnt: &pulumi-mnt
+    type: volume
+    source: pulumi_outputs
+    target: /mnt/pulumi-outputs
+    read_only: true
```

Comment:
thank you - I know some people dislike the yaml <<: stuff - but this is nice reuse :) 

My only thing is that I usually keep the key (`read-only-pulumi-mnt`) and the anchor (`&pulumi-mnt`) the same - but not the end of the world.

Reply:
Yup, good call.